src/Tribe/I18n.php

```diff
@@ -173,8 +173,9 @@ public function get_i18n_strings_for_domains( $strings, $languages, $domains = a
 	 * @return mixed The callback return value, if any.
 	 */
 	protected function with_locale( $locale, callable $do, array $args = [] ) {
-		global $wp_filter;
+		global $wp_filter, $l10n;
 		$locale_filters_backup = isset( $wp_filter['locale'] ) ? $wp_filter['locale'] : [];
+		$l10n_backup = $l10n;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 11 spaces but found 1 space

<sub>hash: 72bfb376ea34fb9ce0dccda4216969ab</sub>